GLAA.Services/Admin/AdminUserPostDataHandler.cs

```diff
@@ -32,6 +39,20 @@ public string Insert(AdminUserViewModel model)
 
             userManager.AddToRoleAsync(user, model.Role).GetAwaiter().GetResult();
 
+            // For more information on how to enable account confirmation and password reset please
+            // visit https://go.microsoft.com/fwlink/?LinkID=532713
+            var code = userManager.GeneratePasswordResetTokenAsync(user).GetAwaiter().GetResult();
+            var callbackUrl = url.Action("ResetPassword", "AccountController", new { userId = user.Id, code = code }, scheme);
```

Comment:
This is cool, didn't know you could set a callbackUrl like this

Reply:
Open source frameworks are great!